app/brief_utils.py

```diff
@@ -1,10 +1,35 @@
-from flask import abort, request
 import json
+
 import rollbar
+from flask import abort, request
 
 from .models import Service
-from .validation import get_validation_errors
 from .service_utils import filter_services
+from .validation import get_sections, get_validation_errors
```

Comment:
dont agree with adding all this code to `brief_utils.py` we want to avoid adding code outside `app/api`. Can it go in a service?